xapian-letor/include/xapian-letor/ranker.h

```diff
@@ -354,6 +354,68 @@ class XAPIAN_VISIBILITY_DEFAULT SVMRanker: public Ranker {
 
 };
 
+class XAPIAN_VISIBILITY_DEFAULT ListMLERanker: public Ranker {
```

Comment:
Put a space either side of the `:` in class inheritance. (If the line gets too long, then wrap before the `:`; see `xapian-core/include/xapian/cluster.h` for examples of both styles).